ovirt_imageio/client/_ui.py

```diff
@@ -15,32 +15,33 @@
 
 class ProgressBar:
 
-    def __init__(self, size=0, output=sys.stdout, step=0.1, width=79,
+    def __init__(self, size=0, output=sys.stdout, step=None, width=79,
```

Comment:
We fill the terminal line (kind of, assuming 80 characters width) so avoid left overs
from previous progress line in the last line:

    "[ 100% ] ...\n                                                 "

If the previous progress line was longer than the last, the extra spaces covers it.

Practically only few spaces can be enough, but filling the entire line is more
robust in case we change the content displayed on the progress line. For example
we may show the current step (e.g. "Preparing for copy", "Copying disk", "Flushing data") if we have several steps.